### DISCUSSION TITLE
dekodemo-door  in Hiddify 🌏

### DISCUSSION THREAD

**Discussion by Gold-IP** [hiddify/Hiddify-Manager#4005] [Q&A سوالات رایج]
لطفا یه روش صحیح معرفی کنین برای ایجاد تانل دکودمودور در هیدیفای.خود اسکریپتی که قرار دادین مشکل داره و به درستی کارنمیکنه.مخصوصن مولتی اینباندش

**Comment by Gold-IP** [comment]
یا اگه روش دیگه ای سراغ دارین که سرعتش بالا باشه معرفی کنین.آیا بانصب x-ui روی سرور ایران و ایجاد اینباند دکودمودور میشه تانل زد به هیدیفای؟

**Reply by ReturnFI** [reply]
سلام.
چه مشکلی داره عزیز؟
در رپازیتوری خود اپ ریلی [ایشو ](https://github.com/hiddify/hiddify-relay/issues) ثبت کنید
